Sources/Vapor/Routing/Parameters+Require.swift

```diff
@@ -20,11 +20,13 @@ extension Parameters {
         where T: LosslessStringConvertible
     {
         guard let stringValue: String = get(name) else {
-            throw Abort(.internalServerError)
+            self.logger.debug("The parameter provided does not exist")
+            throw Abort(.internalServerError, reason: "The parameter provided does not exist")
         }
 
         guard let value = T.init(stringValue) else {
-            throw Abort(.unprocessableEntity)
+            self.logger.debug("The parameter value could not be converted to the required type")
```

Comment:
```suggestion
            self.logger.debug("The parameter \(stringValue) could not be converted to \(T.Type)")
```

(You'll need to check that locally, I think T.Type should compile but worth checking)